app/assets/javascripts/discourse/app/lib/link-mentions.js

```diff
@@ -46,7 +46,7 @@ const found = {};
 const foundGroups = {};
 const mentionableGroups = {};
 const checked = {};
-const cannotSee = [];
+export const cannotSee = {};
```

Comment:
Just an interesting observation but where were calling `cannotSee[username]` in line 55 when `cannotSee` is an Array. I think we fixed that line now that this is an object now.

Reply:
I noticed the bug and decided to just fix it. It's probably not a big deal anyway.